docs/release-notes/next/PT-162445499-multiple-simultaneous-miners.md

```diff
@@ -0,0 +1,2 @@
+- Changes cuckoo miner config and processing to allow multiple simultaneous miners.
+  This impacts mining configuration (changes `miner` param to `miners`).
```

Comment:
I see:
* `edge_bits` moved in common to miners
* `extra_args` made optional
* `extra_args` default changed
* `miner` changed to `miners`

May I suggest:
* Considering to enable old config to still work, though marking relevant sections as deprecated in the schema.
* Add user config test with old config e.g. `apps/aeutils/test/data/epoch_deprecated_miner_config.yaml`
* Add functional test e.g. new system test suite with old config that mines a couple of blocks?

Reply:
It is doable, but will make the code a bit more complex to maintain both old and new configs. For how long would you want to maintain the deprecated config then? 🤔

Reply:
I would expect that 1.2.0 recommends in all docs the new was of specifying miner config and has the deprecation in the user config schema. Then 1.3.0 drops the deprecated config. Cc @dincho @sennui

Reply:
Another option would be to set `"additionalProperties" : true` in `mining` > `cuckoo` section of the config. Then if user deploys the code as it is in the PR, and keeps the old config, she should notice that the default miner is working instead of what she wants, and will adjust config accordingly? So in this aspect it is backwards compatible (node will not crash with old config), but maybe is even more confusing?! I see @lucafavatella 's point, but am not 100% convinced that this is how we should approach it...

@hanssv @dincho thoughts?